CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.0.4 (2023-01-06)
+## 1.0.4 (2023-03-29)
```

Comment:
looks like package.json was already updated in master and I'm just fixing the release date here